rest/data/user.go

```diff
@@ -77,6 +82,24 @@ func (u *DBUserConnector) UpdateSettings(dbUser *user.DBUser, settings user.User
 		settings.Notifications.BuildBreakID = ""
 	}
 
+	var spawnhostSubscriber event.Subscriber
```

Comment:
just an FYI, but I'll clean this up so that there is less duplication in the commit for my other host trigger.